port-mirroring: share pragma pack on struct

tallua opened the pull request:
## 📦 Package Details

**Maintainer:** @mmaraya

**Description:**
For following issue:
https://github.com/openwrt/packages/issues/27394

Found `struct pm_cfg cfg;` is also used on `config.c`.
So I think sharing alignment for struct pm_cfg is better than just moving `getRemoteARP` to `main.c` (which I suggested on issue`)

---

## 🧪 Run Testing Details

- **OpenWrt Version:** 24.10.0
- **OpenWrt Target/Subtarget:** ramips/mt7621
- **OpenWrt Device:** iptime A8004T

---

## ✅ Formalities

- [x] I have reviewed the [CONTRIBUTING.md](../CONTRIBUTING.md) file for detailed contributing guidelines.

### If your PR contains a patch:

- [x] It can be applied using `git am`
- [x] It has been refreshed to avoid offsets, fuzzes, etc., using
  ```bash
  make package/<your-package>/refresh V=s
  ```
- [x] It is structured in a way that it is potentially upstreamable
<sub>(e.g., subject line, commit description, etc.)</sub>
<sub>We must try to upstream patches to reduce maintenance burden.</sub>

GeorgeSapkin commented:
Both commit message and PR title have typos. You need to use your real name in the author and signoff. Commit message could you some more details about why this is needed. The new patch has no header. Where is it coming from and what does it do? Can this fix be merged upstream?

tallua commented:
> Both commit message and PR title have typos. You need to use your real name in the author and signoff. Commit message could use some more details about why this is needed. The new patch has no header. Where is it coming from and what does it do? Can this fix be merged upstream?

Thank you for comment and sorry for the not following contributing.md.
- For header and some description on patch file, I was only looking at the patches on same package. I found that recent PR has commit description referecing original.
- For typo, I fixed on the commit and changed PR title. (also changed title meaning)
- For rean name and email, I'll consider updating my profile for further contribution


I 'll close this PR

BKPepe commented:
> IMHO, I don't know if it's worth maintaining the package

@wehagy Based on your findings, I propose to drop this package from our repo.
